dev/source/docs/sitl-with-airsim.rst

```diff
@@ -19,6 +19,11 @@ Currently, only support for ArduCopter vehicle has been developed in AirSim & Ar
 
 AirSim is an excellent platform for testing and developing systems based on Computer Vision, etc. on simulated vehicles. It's a very feature-rich simulator with detailed environments and APIs (Python, C++, ROS) for collecting data. See the `main Airsim Readme <https://github.com/microsoft/AirSim#welcome-to-airsim>`__ for details and the features available.
 
+A demo of AirSim running with ArduPilot SITL
+
+.. youtube:: 0kE6gc7pn8M
+    :width: 100%
```

Comment:
https://youtu.be/0kE6gc7pn8M
A better video would be with a real RC, but don't have one of those right now
Leaving it upto the maintainers to decide whether to add it or not